csrankings-d.csv

```diff
@@ -526,7 +526,6 @@ David Clark 0001,University College London,http://www0.cs.ucl.ac.uk/people/D.Cla
 David Cordes,The University of Alabama,http://cordes.cs.ua.edu,NOSCHOLARPAGE
 David Corne,Heriot-Watt University,https://www.hw.ac.uk/staff/uk/macs/david-wolfe-corne.htm,6jNgTK4AAAAJ
 David Coyle,University College Dublin,https://people.ucd.ie/d.coyle,9d9bOOUAAAAJ
-David Crandall,Indiana University,http://www.cs.indiana.edu/~djcran,8bQRH5YAAAAJ

```

Comment:
DBLP lists David Crandall 0001 as an alias for David J. Crandall. Please fix. https://dblp.org/pid/c/DavidCrandall.html

Reply:
So this entry should be kept as is?

Reply:
You need both entries.

Reply:
Yes; the latest commit has both entries.